app/models/APIValuationHistory.scala

```diff
@@ -35,6 +36,20 @@ case class APIValuationHistory(
   def capatalise = this.copy(address = address.toUpperCase)
 }
 
+
+
 object APIValuationHistory {
   implicit val formats = Json.format[APIValuationHistory]
+
+  def apply(history: ValuationHistory) :APIValuationHistory =
+    APIValuationHistory(asstRef = history.asstRef,
+    listYear = history.listYear,
+    uarn = history.uarn,
+    effectiveDate = history.effectiveDate.get,
```

Comment:
.get on option.

Reply:
Something I meant to go back and fix like the .get in the frontend. Are you OK for me to change this to option all the way through the journey and handle it in the view on the frontend?

Reply:
Pass option and handle in frontend